addons/mail/static/src/scss/mail_invite_wizard.scss

```diff
@@ -0,0 +1,5 @@
+.o_mail_extra_comments .note-editable {
```

Comment:
Would avoid custom css in classic wizards ... not sure what it solves, but can't you use bootstrap classes (or just skip it entirely) ?

Reply:
Up

Reply:
This particular css helps in reshaping the message textbox, making it exactly 80px in height and adding some padding. I tried using BS classes but the result doesn't seems to be what we want.

Reply:
Would remove border width (should be standard); padding can be achieved using bootstrap classes.

Min height is weird, why would we want that ? But ok that one cannot be done using bs classes I think.